.env.example

```diff
@@ -1,22 +1,31 @@
-# Dev settings
-DEBUG=1
-SECRET_KEY=
-SOCIAL_AUTH_GOOGLE_OAUTH2_KEY=
-SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=
+# Django
+DJANGO_SETTINGS_MODULE=$DJANGO_SETTINGS_MODULE
+DEBUG=$DEBUG
+SECRET_KEY=$SECRET_KEY
 
-# Dev and Prod settings
-ES_PUBLIC_API_KEY=search-7gvgdz431agb5agermmcuvx8
-ES_COURSE_SEARCH_ENDPOINT=https://1761244fa3674318b11f8d5729ffd7a2.app-search.us-east4.gcp.elastic-cloud.com/api/as/v1/engines/uva-courses/search
-ES_INSTRUCTOR_SEARCH_ENDPOINT=https://1761244fa3674318b11f8d5729ffd7a2.app-search.us-east4.gcp.elastic-cloud.com/api/as/v1/engines/uva-instructors/search
+# social-auth-app-django library
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY=$SOCIAL_AUTH_GOOGLE_OAUTH2_KEY
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=$SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET
 
-# For prod settings, specify these vars:
-# DEBUG=0
-# PUBLIC_IPV4=
-# DB_NAME=
-# DB_HOST=
-# DB_USER=
-# DB_PASSWORD=
-# DB_PORT=
-# ES_COURSE_DOCUMENTS_ENDPOINT=
-# ES_INSTRUCTOR_DOCUMENTS_ENDPOINT=
-# ES_PRIVATE_API_KEY=
+# database
+DB_NAME=$DB_NAME
+DB_USER=$DB_USER
+DB_PASSWORD=$DB_PASSWORD
+DB_HOST=$DB_HOST
+DB_PORT=$DB_PORT
+
+# elasticsearch
+ES_PUBLIC_API_KEY=$ES_PUBLIC_API_KEY
+ES_COURSE_SEARCH_ENDPOINT=$ES_COURSE_SEARCH_ENDPOINT
+ES_INSTRUCTOR_SEARCH_ENDPOINT=$ES_INSTRUCTOR_SEARCH_ENDPOINT
+ES_COURSE_DOCUMENTS_ENDPOINT=$ES_COURSE_DOCUMENTS_ENDPOINT
+ES_INSTRUCTOR_DOCUMENTS_ENDPOINT=$ES_INSTRUCTOR_DOCUMENTS_ENDPOINT
+ES_PRIVATE_API_KEY=$ES_PRIVATE_API_KEY
+
+# discord
+DISCORD_URL_BUG=$DISCORD_URL_BUG
+DISCORD_URL_FEEDBACK=$DISCORD_URL_FEEDBACK
+
+# No longer used but
```

Comment:
Is it safe to just delete these?

Reply:
If you look at the code, they're not used anywhere because they're all commented out. Normally I would delete these, but I was reminded of `# PYTHONUNBUFFERED = 1` in `Dockerfile`, which I removed because it was a comment but turned out to be helpful. (That's #299 btw.)